frontend/src/components/AddRecipeForm.js

```diff
@@ -190,8 +231,80 @@ const AddRecipeForm = () => {
               </button>
             </div>
           </div>
+          <div className="d-flex align-items-center">
```

Comment:
Az egész hozzavaló felvétel egység lehetne egy külön komponens, ahol történik az API kérés is a hozzáadásra. Lehetne egy onIngredientAdded eseménye, aminek hatására frissítitek a listát.